pkgs/libresoc/openpower-isa.nix

```diff
@@ -37,6 +37,9 @@ with python39Packages;
     patches = [
       # setup.py uses @git syntax for version specifiers, prevent setuptools from attempting to clone
       ./use-vendored-git-dependencies.patch
+      # patch out hack to create empty directory in output build
+      # see postInstall notes for complement
+      ./remove-gitignore-check.patch
```

Comment:
LibreSOC wants to create an empty directory as part of its build at `openpower/decoder/isa/generated`
pip uses MANIFEST.in to build a wheel, but will ignore empty directories by default
To get around this, LibreSOC included a `.gitignore` so the directory was actually not empty
However, nix by default strips out git metadata, including `.gitignore` files. So now the directory is the empty again and will not get copied to the output.
One solution is to make nix actually pay attention to the git metadata, but doing that would also clone a bunch of submodules in this project, which then defeats the purpose of using all the vendored nix dependencies.
So instead, this patch removes the code that was checking in the `.gitignore`. And we `mkdir` in `postInstall` instead